packages/foam-vscode/package.json

```diff
@@ -514,7 +514,20 @@
         "foam.preview.embedNoteInContainer": {
           "type": "boolean",
           "default": true,
-          "description": "Wrap embedded notes in a container when displayed in preview panel"
+          "description": "Wrap embedded notes in a container when displayed in preview panel",
+          "deprecationMessage": "*DEPRECATED* use foam.preview.embedNoteType instead."
+        },
+        "foam.preview.embedNoteType": {
+          "type": "string",
+          "default": "full-card",
+          "enum": [
+            "full-inline",
+            "full-card"
```

Comment:
decided to save `content-inline` and `content-card` enum options for when we actually add their implementation since it would be confusing if the options didn't do anything.

Reply:
Yes, if we are not adding the implementation we shouldn't have the options 👍